src/business.ts

```diff
@@ -0,0 +1,121 @@
+import type { Marqeta, MarqetaOptions, MarqetaError } from './'
+
+export interface BusinessIdentifications {
+  type: string;
+  value: string;
+}
+
+export interface IncorporationType {
+  incorporationType: string;
+  stateOfIncorporation: string;
+}
+
+export interface Address {
+  address1: string;
+  address2: string;
+  city: string;
+  state: string;
+  postalCode: string;
+  country: string;
+}
+
+export interface Personal {
+  firstName: string;
+  lastName: string;
+  home: Address;
+}
+
+export interface Meta {
+  name: string;
+}
+
+export interface Business {
+  token?: string,
+  createdTime: string,
+  metadata: Meta,
+  accountHolderGroupToken: string,
+  lastModifiedTime: string,
+  status: string,
+  attestorName: string;
+  attestationConsent: string;
+  attestationDate: string;
+  businessNameLegal: string;
+  businessNameDba: string;
+  identifications: BusinessIdentifications[];
+  incorporation: IncorporationType;
+  dateEstablished: string;
+  proprietorOrOfficer: Personal;
+}
+
+export interface BusinessList {
+  count: bigint;
+  startIndex: bigint;
+  endIndex: bigint;
+  isMore: boolean;
+  data: Business[];
+}
```

Comment:
Typically a blank line is between the interface definitions and the class definition.